apis/nucleus/src/utils/style/styling-props.js

```diff
@@ -67,9 +68,9 @@ export function resolveBgImage(bgComp, app) {
   if (bgImageDef) {
     let url = '';
     if (bgImageDef.mode === 'media' || bgComp.useImage === 'media') {
-      url = bgImageDef?.mediaUrl?.qStaticContentUrl?.qUrl
-        ? decodeURIComponent(bgImageDef.mediaUrl.qStaticContentUrl.qUrl)
-        : undefined;
+      const urlObj = bgImageDef?.mediaUrl || bgImageDef?.url;
```

Comment:
Same as the comment above but for `url` here. Any specific reason to why we add yet another prop for the same purpose.